src/alto-ui/Form/TagInput/hooks/useTagInputStateAndRefs.js

```diff
@@ -0,0 +1,58 @@
+import { useRef, useEffect } from 'react';
+
+import useResettableState from '../../../hooks/useResettableState';
+import useFocusInOut from '../../../hooks/useFocusInOut';
+import usePosition from './usePosition';
+
+export default function useTagInputStateAndRefs(tags, ref) {
+  const mainRefDefault = useRef();
+  const mainRef = ref || mainRefDefault;
+  const inputRef = useRef();
+  const valueRef = useRef();
+  const isActive = useFocusInOut(null, mainRef);
+
+  const state = {
+    value: useResettableState(''),
+    selection: useResettableState([]),
+    position: usePosition(tags),
+    isActive,
+  };
+
+  const [value, , resetValue] = state.value;
```

Comment:
Why not return object from `useResettableState` and make reference with by simply `.` 
You can avoid that kind of strange destructurization assigment when you must put empty `,` to skip value.  ?

Reply:
very good point, I am not a big fan also. I tried to keep the React style. But using an object instead was actually my first dev. Check this commit and tell me if you prefer. https://github.com/beqom/alto/pull/564/commits/aa28db42331efbdb0310bf440a17414999a5b2fa

Reply:
Yep, I know that the main point to use array is that you can name variables as you want but maybe instead writing 
```
  const [position, , resetPosition] = state.position;
```
we can use aliases
```
const {get: position, reset: resetSelecion} = state.position;
```
And this is more obvious than some indexes skipping

Reply:
we shared the same analysis with @NicolasMartos . I will do the second one ;)